docs/docsite/rst/reference_appendices/module_utils.rst

```diff
@@ -5,65 +5,25 @@
 Ansible Reference: Module Utilities
 ***************************************************************
 
-This page documents the available utilities called with ``ansible.module_utils.util_name``.
+This page documents utilities intended to be helpful when writing
+Ansible modules in Python.
 
-Generic
---------
-.. glossary::
-
-    .. _ansible.module_utils.debug
-    debug
-    .. _ansible.module_utils.url
-    url
-    .. _ansible.module_utils.log
-    log
-    .. _ansible.module_utils.no_log
-    no_log
-    .. _Ansible.get_bin_path
-    Ansible.get_bin_path
-
-
-.. _AnsibleModule:
-.. _ansible.module_utils.basic.AnsibleModule:
 
 AnsibleModule
 --------------
-.. glossary::
 
-    ansible.module_utils.basic.AnsibleModule
-      Utilities in module_utils.basic.AnsibleModule apply to all module types
+This functionality is available via ``from ansible.module_utils.basic import AnsibleModule``.
```

Comment:
```suggestion
To use this functionality, include ``from ansible.module_utils.basic import AnsibleModule`` in your module.
```

Reply:
For the record: I had this in two places, fixed in both as of 48e8c74.